test/fuzzing/README.md

```diff
@@ -0,0 +1,19 @@
+# Fuzzing Linkerd
+
+The scripting setup for fuzzing is used by
+[google/oss-fuzz](https://github.com/google/oss-fuzz) which performs continuous
+fuzzing for the Linkerd project.
+
+The fuzzing configuration for Linkerd is located in the
+[linkerd2](https://github.com/google/oss-fuzz/tree/master/projects/linkerd2)
+project which handles the docker build and execution of the fuzzers.
+
+### oss-fuzz File Setup
+
+- [Dockerfile](https://github.com/google/oss-fuzz/blob/master/projects/linkerd2/Dockerfile)
```

Comment:
Having more concrete instructions for how to run this would be nice

Reply:
a0ad5d8 adds documentation links for oss-fuzz and how to run it locally.